gen/typedef_test.go

```diff
@@ -23,74 +23,87 @@ package gen
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
import group ordering we follow is:

- standard library
- local library imports
- external library imports